service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/dataproccluster/startup.sh

```diff
@@ -85,6 +85,8 @@ readonly TERRA_BOOT_SERVICE_OUTPUT_FILE="${USER_TERRA_CONFIG_DIR}/boot-output.tx
 
 readonly JUPYTER_SERVICE_NAME="jupyter.service"
 readonly JUPYTER_SERVICE="/etc/systemd/system/${JUPYTER_SERVICE_NAME}"
+readonly ETC_JUPYTER="/etc/jupyter"
```

Comment:
If you don't have a second use for this particular variable, I'd say just have JUPYTER_CONFIG be the full path:

```
readonly JUPYTER_CONFIG="/etc/jupyter/jupyter_notebook_config.py"
```